assets/prometheus-k8s/rules.yaml

```diff
@@ -710,7 +710,7 @@ spec:
       annotations:
         message: Job {{ $labels.namespace }}/{{ $labels.job_name }} failed to complete.
       expr: |
-        kube_job_status_failed{namespace=~"(openshift-.*|kube-.*|default|logging)",job="kube-state-metrics"}  > 0
+        kube_job_failed{namespace=~"(openshift-.*|kube-.*|default|logging)",job="kube-state-metrics"}  > 0
```

Comment:
this doesn't have any side effect for us?

Reply:
I didn't find any.

Reply:
I found something in openshift-azure, but no clue if these files are used, looks generated https://github.com/openshift/openshift-azure/blob/48f07ce4f72307ff4397eb5d2d9352fbee6597b6/pkg/entrypoint/metricsbridge/metrics-bridge-example-config.yaml#L46

Reply:
Yeah, that looks generated. Also it is an example and has more problems as we already changed some other metrics from that file.